frontend/tailwind.css

```diff
@@ -2,6 +2,54 @@
 @tailwind base;
 /* purgecss end ignore */
 
+html {
+    @apply text-base text-font-primary;
+}
+
+@screen lg {
+    html {
+        @apply text-lg-base;
+    }
+}
+
+body {
+    @apply text-body;
+}
+
+a {
+    @apply text-link-primary;
```

Comment:
what happens if a link on page x has another style?

Reply:
we have 2 options:
1. Create more default styles, eg a.btn-blue, and apply the class
2. Use specific TW styles on the individual links

Reply:
Agreed, I know it's a bit against functional CSS to have a style applied globally to all `a` tags but I think we can handle specific links with different styles with any of those options. Same for headings below.

Reply:
Yes. We just need to think about when to use each option. As a first stab I think we should define defaults for some basic typography and maybe some html tags, like <a> (but even this can cause problems as an a in body text will be completely different from an a in a menu for example). We prob want to use these defaults very sparingly.

Then the additional utility classes can be used for elements which are repeated but without a single php template (if a php template exists then just use that instead with functional classes).

Reply:
I thought some more about this and we can use more functional CSS by applying classes to the core blocks (as well as the custom ones). I set this up in Lean as some config like this so it's easy to configure:
![image](https://user-images.githubusercontent.com/5068959/71695762-b721ea00-2d78-11ea-9a4d-9b76a5346611.png)